examples/cloud-composer-examples/composer_http_post_example/ephemeral_dataproc_spark_dag.py

```diff
@@ -86,31 +78,33 @@
         # ds_nodash is an airflow macro for "[Execution] Date string no dashes"
         # in YYYYMMDD format. See docs https://airflow.apache.org/code.html?highlight=macros#macros
         cluster_name='ephemeral-spark-cluster-{{ ds_nodash }}',
+        image_version='1.5-debian10',
         num_workers=2,
         storage_bucket=Variable.get('dataproc_bucket'),
         zone=Variable.get('gce_zone'))
 
     # Submit the PySpark job.
     submit_pyspark = DataProcPySparkOperator(
         task_id='run_dataproc_pyspark',
-        main=PYSPARK_JOB,
+        main='gs://' + Variable.get('gcs_bucket') +
```

Comment:
Note: frustratingly this field is not a template field (in default composer version of airflow 1.10.3)

Reply:
This retrieves the variable value at dag parse time rather than runtime.
This is sufficient for the codelab but not ideal.

Reply:
Is this fixed in the master branch?

Reply:
@mik-laj if you mean is this fixed in airflow master, no.
[`DataprocSubmitPySparkJobOperator.template_fields`](
https://github.com/apache/airflow/blob/master/airflow/providers/google/cloud/operators/dataproc.py#L1259) does not contain main.

Opened a new issue https://github.com/apache/airflow/issues/9146